dll/win32/shell32/shellmenu/CStartMenu.cpp

```diff
@@ -149,6 +153,46 @@ class CShellMenuCallback :
         return S_OK;
     }
 
+    HMENU CreateRecentMenu() const
+    {
+        HMENU hMenu = ::CreateMenu();
+
+        BOOL bExpandMyDocuments = FALSE; /* FIXME: Get value from registry */
```

Comment:
Where (which property page) should we have these parameters set ? Why no already putting the registry reading code and having another PR on settings along with this one ?

Reply:
That will be in `explorer.exe`. That issue is beyond this PR.

Reply:
OK, once this one is merge could we discuss together in order to get the complementary PR done ? (I can help / take care of it)

Reply:
You can do it later.

Reply:
ok